index.js

```diff
@@ -65,6 +65,19 @@ cuid.slug = function slug () {
     counter + print + random;
 };
 
+cuid.isCuid = function isCuid (stringToCheck) {
+  if (typeof stringToCheck !== 'string') return false;
+  if (stringToCheck.startsWith('c') && stringToCheck.length === 25) return true;
```

Comment:
String length is not guaranteed by the algorithm.

Reply:
@ericelliott Should I remove the length check altogether?

Reply:
Yes.

Reply:
@ericelliott Fixed in 5f52a9b.